### ISSUE TITLE
App Icon not working in Fedora Overview/Dock

### ISSUE THREAD

**Issue by meetjakewood** [ciderapp/Cider-2#1252]
### Preflight Checklist

- [x] I agree to follow the [Code of Conduct](https://cider.sh/legal/conduct) that this project adheres to.
- [x] I have searched the [issue tracker](https://github.com/ciderapp/Cider-2/issues) for a bug report that matches the one I want to file, without success.
- [x] I have confirmed that no [support FAQ](https://cider.sh/docs/client/faqs) resolves my issue.
- [x] I have checked that the correct behavior occurs on [Apple Music Web](https://music.apple.com) (if applicable).
- [x] I have verified that my issue is with a [supported version](https://github.com/ciderapp/Cider-2/blob/main/SECURITY.md) and that it is with the desktop app (Support for other areas can be found in our Discord / on the appropriate repo).

### How did you download Cider?

Taproom

### App Information

{"buildInfo":{"version":"3.1.5","commit":"53b47bd4a8d9e643b9d68fc840bd43f142dc1cca","branch":"main","buildDate":"1758927080767","changelogUrl":"3.1.0-release","codeName":"Gala"},"clientInfo":{"client":"genten","env":"production","arch":"x64","platform":"linux","platformFriendly":"Linux","build":"desktop","clientVersion":"310","capi":"kelvah6kh6co2w3dui8e9c62","subversion":"0"}}

### Operating System

Linux

### OS Version

Fedora Workstation

### How would you describe your issue?

Accessibility

### Describe the Bug

The app icon on a fresh install (3.1.5) is not appear in the overview and dock for Fedora Workstation.

<img width="860" height="100" alt="Image" src="https://github.com/user-attachments/assets/46bba379-2b5e-4433-bba9-36200c5784b7" />

### Steps to Reproduce

1. Install Cider fresh via AppImage on Fedora Workstation
2. Install using AppImageLauncher
3. Open Cider
4. No app icon (changing versions doesn't help)

**Comment by ChriFo** [comment]
Here’s a short workaround for anyone seeing a second icon in GNOME when launching Cider from the dock:

The issue seems to be caused by a missing or incorrect StartupWMClass in the .desktop file. You can fix it locally like this:

```
cp /usr/share/applications/cider.desktop ~/.local/share/applications/
nano ~/.local/share/applications/cider.desktop
```

Add or change this line in the [Desktop Entry] section:

```
StartupWMClass=cider
``` 

Then log out and log back in.

After that, remove the old Cider icon from GNOME favorites/dock, start Cider again from the application launcher, and pin the correct icon again.

This fixed the duplicate dock icon issue for me on Fedora GNOME.
